pyscriptjs/src/utils.ts

```diff
@@ -37,4 +37,19 @@ function guidGenerator(): string {
     return S4() + S4() + '-' + S4() + '-' + S4() + '-' + S4() + '-' + S4() + S4() + S4();
 }
 
-export { addClasses, getLastPath, ltrim, htmlDecode, guidGenerator };
+/*
+ *  Display a page-wide error message to show that something has gone wrong with
+ *  PyScript or Pyodide during loading. Probably not be used for issues that occur within
+ *  Python scripts, since stderr can be routed to somewhere in the DOM
+*/
+function showError(msg: string): void {
+    let warning = document.createElement("div")
+    warning.style.backgroundColor = "LightCoral";
+    warning.style.alignContent = "center";
+    warning.style.margin = "4px";
+    warning.style.padding = "4px";
+    warning.innerHTML = msg;
+    document.body.prepend(warning);
+}
```

Comment:
this is a great start! Ideally, it'd be great to have a global error dialog/div that can be used from the user application on the Python side as well but should be an evolution of these.